python/ray/util/client/common.py

```diff
@@ -532,7 +532,7 @@ def __init__(
         self,
         actor_handle: ClientActorHandle,
         method_name: str,
-        num_returns: int,
```

Comment:
Ray client doesn't support streaming yet so we can revert this change.

Reply:
should now be addressed